SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/AbstractLockingPolicy.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "SpatialConstants.h"
+
+#include "GameFramework/Actor.h"
+
+#include "AbstractLockingPolicy.generated.h"
+
+UCLASS(abstract)
+class SPATIALGDK_API UAbstractLockingPolicy : public UObject
+{
+	GENERATED_BODY()
+
+public:
+	virtual ActorLockToken AcquireLock(AActor* Actor, FString LockName = "") PURE_VIRTUAL(UAbstractLockingPolicy::AcquireLock, return SpatialConstants::INVALID_ACTOR_LOCK_TOKEN;);
+	virtual void ReleaseLock(ActorLockToken) PURE_VIRTUAL(UAbstractLockingPolicy::ReleaseLock, return;);
```

Comment:
Any reason why this parameter is not given a name?
```suggestion
	virtual void ReleaseLock(ActorLockToken Token) PURE_VIRTUAL(UAbstractLockingPolicy::ReleaseLock, return;);
```